### PR TITLE
Update notes.php

### PR DESCRIPTION
**Pull request by itsJarrett**
Update notes.php to enable the search of players by their playerid or uid.

### CODE REVIEW

**Comment by Jason2605** [comment]
Don't search by playerid, as that has changed in v5.0 of AL

File: notes.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 if (isset($_POST['search'])) {
     $valuetosearch = $_POST['SearchValue'];
-    $sqlget = "SELECT * FROM players WHERE CONCAT (`name`) LIKE '%".$valuetosearch."%'";
+    $sqlget = "SELECT * FROM players WHERE CONCAT (`name`,`playerid`,`uid`) LIKE '%".$valuetosearch."%'";
```
Comment (Jason2605) [inline notes.php:21]: Down search by playerid, or atleast have checks for pid too. See the players page.

**Comment by itsJarrett** [comment]
Added support per 91da8c0.

**Comment by itsJarrett** [comment]
Looked clean on my screen, I suppose it parsed differently when I commited it.

**Comment by Jason2605** [comment]
Have you tested this works after the changes.

File: notes.php [hunk 2]
```diff
@@ -65,9 +65,9 @@
             <table class="table table-striped" style = "margin-top: -10px">
               <thead>
                 <tr>
+                    <th>ID</th>
```
Comment (Jason2605) [inline notes.php]: You love that tab key

Comment (itsJarrett) [reply notes.php]: https://gyazo.com/a4c108a830574a787a26cbc47e89b904 

Must be PHPStorm,  didn't press tab at all >_>

Comment (Jason2605) [reply notes.php]: Just make sure you use 4 spaces for where a tab would be.

**Review by Jason2605 — CHANGES_REQUESTED** [review]
Tabs.

**Comment by itsJarrett** [comment]
Reconfigured my IDE to change from tabs to spaces.

**Comment by Jason2605** [comment]
Much better.

**Review by Jason2605 — APPROVED** [review]

**Pull request merged**